### PR TITLE
Documentation updates

### PR DESCRIPTION
**Pull request by aaarendt**
Some small updates to the README file and docstring to highlight units of snow depth and the variables returned from the API.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,7 +38,16 @@ mtnhubsnow.snow_data(
 - box -- Bounding box to restrict results, specified as dictionary with items `latmax`, `lonmax`, `latmin`, `lonmin`
 - filter -- Flag indicating whether entries with no snow depth data should be filtered out.
 
-*Returns* a Pandas dataframe with observation timestamp in UTC, latitude and longitude coordinates, and snow depth in meters.
+*Returns:* 
+A Pandas dataframe with the following columns:
+- `id` -- unique identifier for each observation
+- `datetime_utc` -- timestamp in UTC
+- `latitude`
+- `longitude`
```
Comment (emiliom) [inline README.md]: Add something like " -- Longitude in decimal degrees"? And same for latitude

**Comment by emiliom** [comment]
Thanks! But note that you wrote in README.md and the docstring that snow_depth is in cm. It's in **meters** (see the original README.md text). Otherwise, it looks great.

**Comment by aaarendt** [comment]
Do you know where the units are documented on the MH side? I'm comparing data to snodas and am almost certain what this API returns is snow depth in cm.

**Comment by emiliom** [comment]
You're right! I believe I wrote meters in the README way back, but I can't remember how I arrived at that conclusion. Looking back at the MH snow_depth values, you're absolutely right, they can't be in meters. I also looked through the CSO database import code that reads from MH, and everything points to cm.

I'm glad you're paying close attention.

Please merge the PR if you think it's ready. I don't have any other comments myself.

**Comment by emiliom** [comment]
> Do you know where the units are documented on the MH side? 

No. I only have a Word document you sent me quite a while ago that refers to the MH web service at a low level. It doesn't discuss the snow depth data.

**Pull request merged**